### PR TITLE
Add mindrage/Manticore-13B-Chat-Pyg-Guanaco-GGML

### PR DESCRIPTION
**Pull request by lewissvg**

### CODE REVIEW

File: models/manticore-13b-chat-pyg-guanaco.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "_descriptorVersion": "0.0.1",
+  "datePublished": "2023-06-12T11:16:39.000Z",
+  "name": "Manticore 13B Chat Pyg Guanaco",
+  "description": "Manticore-13b-Chat-Pyg by openaccess-ai-collective with the Guanaco 13b qLoRa by TimDettmers applied through Monero, quantized by mindrage, uncensored",
+  "author": {
+    "name": "mindrage",
+    "url": "https://huggingface.co/mindrage",
+    "blurb": "The model seems to have noticeably benefited from further augmentation with the Guanaco qLora. Its capabilities seem broad, even compared with other Wizard or Manticore models, with expected weaknesses at coding. It is very good at in-context-learning and (in its class) reasoning. It both follows instructions well, and can be used as a chatbot. Refreshingly, it does not seem to insist on aggressively sticking to narratives to justify formerly hallucinated output as much as similar models. It's output seems... eerily smart at times. I believe the model is fully unrestricted/uncensored and will generally not berate."
```
Comment (yagil) [inline models/manticore-13b-chat-pyg-guanaco.json:9]: `blurb` is intended to be about the author

File: models/manticore-13b-chat-pyg-guanaco.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
+{
+  "_descriptorVersion": "0.0.1",
+  "datePublished": "2023-06-12T11:16:39.000Z",
+  "name": "Manticore 13B Chat Pyg Guanaco",
+  "description": "Manticore-13b-Chat-Pyg by openaccess-ai-collective with the Guanaco 13b qLoRa by TimDettmers applied through Monero, quantized by mindrage, uncensored",
```
Comment (yagil) [inline models/manticore-13b-chat-pyg-guanaco.json:5]: Here's a description field from another file:

```json
"description": "This qlora fine-tuned 13b parameter LlaMa model uses synthetic training data created via github.com/jondurbin/airoboros. It extends version 1.1, introducing thousands of new training data and an update for \"PLAINFORMAT\" to print code without backticks or explanations. The dataset, available online, focuses on coding, math/reasoning, trivia, role playing, multiple choice, fill-in-the-blank, context-obedient question answering, theory of mind, and general topics. The model was fine-tuned with a qlora fork, updated to use a modified vicuna template compatible with 7b/13b versions. The format involves a preamble/system prompt, followed by \"USER: [prompt] ASSISTANT: \", with the prompt allowing for multiple lines and spaces.",
```
A few things to try to replicate here: 
- about 100 words or less
- introduces the model on a high level
- mentions purpose / motivation, training datasets, what it was trained for / what it's "good at"
- mentions expected prompt format

GPT4 can help with this if you provide snippets from the hugging face page

File: models/manticore-13b-chat-pyg-guanaco.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  "author": {
```
Comment (yagil) [inline models/manticore-13b-chat-pyg-guanaco.json:6]: > Manticore-13b-Chat-Pyg by [openaccess-ai-collective](https://huggingface.co/openaccess-ai-collective/manticore-13b-chat-pyg) with the Guanaco 13b qLoRa by [TimDettmers](https://huggingface.co/timdettmers/guanaco-13b) applied through [Monero](https://huggingface.co/Monero/Manticore-13b-Chat-Pyg-Guanaco), quantized by [mindrage](https://huggingface.co/mindrage)

it sounds like `mindrage` is the "publisher", not the author

File: models/manticore-13b-chat-pyg-guanaco.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        "sha256checksum": "ea266b52d6080e7df1b8d98ea290d0b4e261dc9cfe5fc9169abcef0c154831e5",
+        "publisher": {
+          "name": "mindrage",
+          "socialUrl": "https://github.com/mindrages"
+        },
+        "respository": "mindrage/Manticore-13B-Chat-Pyg-Guanaco-GGML",
+        "repositoryUrl": "https://huggingface.co/mindrage/Manticore-13B-Chat-Pyg-Guanaco-GGML"
+      },
+      {
+        "name": "Manticore-13B.ggmlv3.q6_K.bin",
```
Comment (yagil) [inline models/manticore-13b-chat-pyg-guanaco.json:43]: Name here doesn't match the rest of the fields (`Manticore-13B-Chat-Pyg-Guanaco-GGML-q5_K_S.bin` in the `highlighted` part as well as in the `url` below

**Comment by yagil** [comment]
Thanks a lot for opening this PR @lewissvg. Left you some comments

**Comment by lewissvg** [comment]
Updated based on your comments, should be a lot better now @yagil :)

File: models/manticore-13b-chat-pyg-guanaco.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  "name": "Manticore 13B Chat Pyg Guanaco",
+  "description": "The model, augmented with Guanaco qLora, shows broad capabilities compared to other models like Wizard or Manticore. It excels in in-context learning and reasoning but may have weaknesses in coding. The model follows instructions, works as a chatbot, and produces intelligent responses. It accepts various prompting styles, including the ###-Variant. The model is generally unrestricted and doesn't berate users. Recommended settings include low temperature, low diversity, and slight repetition penalty.",
+  "author": {
+    "name": "Open Access AI Collective",
+    "url": "https://huggingface.co/openaccess-ai-collective/",
+    "blurb": "<blurb needed>"
+  },
+  "numParameters": "13B",
+  "resources": {
+    "canonicalUrl": "https://huggingface.co/mindrage/Manticore-13B-Chat-Pyg-Guanaco-GGML",
```
Comment (yagil) [inline models/manticore-13b-chat-pyg-guanaco.json:13]: The intention is for this to point to the original non-quantized model. Apologies for the lack of documentation right now

Comment (lewissvg) [reply models/manticore-13b-chat-pyg-guanaco.json:13]: Ohh, no worries at all! I should have looked at the other files better 😅 Will update

File: models/manticore-13b-chat-pyg-guanaco.json [hunk 6]
```diff
@@ -0,0 +1,58 @@
+{
+  "_descriptorVersion": "0.0.1",
+  "datePublished": "2023-06-12T11:16:39.000Z",
+  "name": "Manticore 13B Chat Pyg Guanaco",
+  "description": "The model, augmented with Guanaco qLora, shows broad capabilities compared to other models like Wizard or Manticore. It excels in in-context learning and reasoning but may have weaknesses in coding. The model follows instructions, works as a chatbot, and produces intelligent responses. It accepts various prompting styles, including the ###-Variant. The model is generally unrestricted and doesn't berate users. Recommended settings include low temperature, low diversity, and slight repetition penalty.",
+  "author": {
+    "name": "Open Access AI Collective",
+    "url": "https://huggingface.co/openaccess-ai-collective/",
+    "blurb": "<blurb needed>"
```
Comment (yagil) [inline models/manticore-13b-chat-pyg-guanaco.json:9]: I know I have this in another place for `openaccess-ai-collective` but let's leave it empty `""` for this one

**Comment by yagil** [comment]
Looks great! 2 remaining comments: https://github.com/lmstudio-ai/model-catalog/pull/7#discussion_r1236023304 https://github.com/lmstudio-ai/model-catalog/pull/7#discussion_r1236020978 and good to go

**Review by yagil — APPROVED** [review]

**Pull request merged**